plotting/plotdataMC.py

```diff
@@ -309,13 +316,13 @@
         )
         hmc = collated["mc"][discr][noSF_axis]
         ax.stairs(
-            values=hmc.values() + np.sqrt(hmc.values()),
-            baseline=hmc.values() - np.sqrt(hmc.values()),
+            values=hmc.values() + np.sqrt(hmc.variance()),
+            baseline=hmc.values() - np.sqrt(hmc.variance()),
```

Comment:
Sorry not make it clear in the description, this actually relates to the issue here `plotting : fix error band on upper panel` 

This is found with using low stat sample(W+jets) in the plot. The error of the weighted events should be `sqrt(sum(w^2))` instead of` sqrt(sum(N))`,

Reply:
Alright, thanks for confirming / clarifying.
Then the last question on that matter is whether L266-267 needs to be changed as well?

Reply:
Thanks @AnnikaStein  for this nice catch, should be modified and included in following commit